steering_committee_guidelines.rst

```diff
@@ -0,0 +1,87 @@
+
+Expectations of a Committee Member:
+===================================
+
+As a steering committee member, you agree to:
+
+1. Abide by the Ansible Community Code of Conduct.
+2. Join and participate in weekly meetings and asynchronous discussions.
+3. Not promote corporate or individual agenda during the decision making
+   process.
+4. Review other proposals and provide feedback.
+5. Represent the needs of the community.
+6. Engage with the community in a professional and positive manner.
+7. Encourage community members to express their opinion.
+
+Joining the Steering Committee:
+===============================
+
+Eligibility:
+~~~~~~~~~~~~
+
+1. Have a wide knowledge of Ansible and its related projects.
+2. Active Contributions to the Ansible project.
+3. Voicing out opinions whenever necessary.
+
+Process:
+~~~~~~~~
+
+1. Any community member may nominate someone/themselves for committee
+   membership at any time.
+2. Only the present Steering committee members may vote on selecting a
+   new member.
+3. Nominees must receive a majority vote from the present members to be
```

Comment:
What do you mean by 'present'?

I think it might be better to define this in an async fashion right away, like every member needs to be made aware in advance of an application, which can then say be discussed for a certain amount of time, and then there's more time to cast votes. I have no idea what is a good timeframe for such votes, but I would suggest it should be at least a couple of weeks (say 2-3) between nomination and closing the vote, to avoid committee members to go on vacation and finding out when they return that suddenly there are 20 new members they never have heard of :)

Reply:
I would suggest removing `present` for now, fix other minor logical / wording things, and merging the docs in their basic versions ASAP.
Then we could elaborate on different aspects of the committee by separate PRs (I'm planning to do it ASAP), i.e. PR per aspect. Otherwise, this will probably be never merged:)